/**:
  ros__parameters:
    image_topic: "/kitti/camera/color/left/image_raw"   # Image topic to subscribe to
    depth_weights_file: "/weights/rt_monodepth_full.engine" # Path to depth estimation file
    camera_frame: "cam2_link"
    base_frame: "base_link"
    camera_image_height: 640
    camera_image_width: 640
    fx: 721.5377
    fy: 721.5377
    cx: 320.0
    cy: 96.0
